dev-libs/libusb-compat/libusb_compat-0.1.7.recipe

```diff
@@ -5,13 +5,13 @@ libusb-compat is a wrapper library for software that use the 0.1 API."
 HOMEPAGE="https://libusb.info/"
 COPYRIGHT="2008 Daniel Drake"
 LICENSE="GNU LGPL v2"
-REVISION="7"
-SOURCE_URI="ftp://sourceforge.mirrorservice.org/sites/downloads.sourceforge.net/l/li/libusb/libusb-compat-0.1/libusb-compat-0.1.5/libusb-compat-0.1.5.tar.bz2"
-CHECKSUM_SHA256="404ef4b6b324be79ac1bfb3d839eac860fbc929e6acb1ef88793a6ea328bc55a"
-SOURCE_DIR="libusb-compat-$portVersion"
-PATCHES="libusb_compat-0.1.5.patchset"
+REVISION="1"
+SOURCE_URI="https://github.com/libusb/libusb-compat-0.1/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d9ca58ec05ba5a29646a7549b9c2c02475a7ed86637337a4ee35b5d559de6e0d"
+SOURCE_DIR="libusb-compat-0.1-$portVersion"
+PATCHES="libusb_compat-$portVersion.patchset"
 
-ARCHITECTURES="x86 x86_64 x86_gcc2"
```

Comment:
didn't check on x86_gcc2, right? ;)

Reply:
Only, x86 (GCC8) - as I see the majority (99%) of Haikuports dependencies not using x86_gcc2 libusb*.

Reply:
If it's enabled it's best to make sure it builds